src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -116,6 +114,7 @@ def spring_cloud_update(cmd, client, resource_group, name, app_insights_key=None
 
 
 def spring_cloud_delete(cmd, client, resource_group, name, no_wait=False):
+    logger.warning("Stop using Azure Spring Cloud? We appreciate your feedback: aka.ms/springclouddeletesurvey")
```

Comment:
Use the full `https://aka.ms/springclouddeletesurvey` so the link can be recognized and clicked from shell.

Reply:
@fengzhou-msft Good idea, have updated it, thanks